exercises/practice/spiral-matrix/.docs/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+Given the size, return a square matrix of numbers in spiral order.
+
+The matrix should be filled with natural numbers, starting from 1 in the top-left corner, increasing in an inward, clockwise spiral order, like these examples:
```

Comment:
```suggestion
The matrix should be filled with natural numbers, starting from 1 in the top-left corner, increasing in an inward, clockwise spiral order.
```

Because a colon and then header level (section change) does not quite make sense to me.

Reply:
@kotp this is the auto-generated text from `bin/configlet create --practice-exercise spiral-matrix`, should this be changed upstream in this case?

Reply:
It probably does.